app/next-client-app/types/concepts.ts

```diff
@@ -0,0 +1,13 @@
+interface Concept {
```

Comment:
I would move this to the `omop` types file also.

Reply:
This is already moved